### ISSUE TITLE
Accessibility | Chip component - not possible to update accessibility-title for inner component "checkmark" icon or hide for screen readers

### ISSUE THREAD

**Issue by leoAnastasi** [telekom/scale#2446]
In Chip in "selected" state in HTML we have checkmark icon with accessibility title "success"

`<scale-chip selected>Label</scale-chip>`

`<scale-icon-action-checkmark accessibility-title="success" size="16" selected="" class="hydrated" style="display: inline-flex;">...</scale-icon-action-checkmark>`

Аnd from [documentation](https://telekom.github.io/scale/?path=/docs/components-chip--standard) there is no way to change this inner icon outside, so we cannot translate title for our blind users or hide this element for screen readers.

Hint from our accessibility team:
"Layout graphics that are used for decorative purposes only should be marked with an empty alt attribute in the case of img tags. Alternatively or additionally, the affected elements can be hidden for screen reader users using the aria-hidden attribute"
